src/opm/parser/eclipse/EclipseState/Schedule/ActionX.cpp

```diff
@@ -65,13 +65,10 @@ void ActionX::addKeyword(const DeckKeyword& kw) {
 bool ActionX::eval(std::time_t sim_time, const ActionContext& context) {
     if (!this->ready(sim_time))
         return false;
-    bool result = true;
 
```

Comment:
This construction is a placeholder for code which will come; and can return `false` - so I would prefer to leave it as is.